velox/common/base/SuccinctPrinter.h

```diff
@@ -25,6 +25,12 @@ namespace facebook::velox {
 /// The default precision is 2 decimal digits.
 std::string succinctNanos(uint64_t duration, int precision = 2);
 
+/// Match the input duration in microseconds to the most appropriate unit and
+/// return a string value. Possible units are microseconds(us),
```

Comment:
What does "Possible units are..." refer to?

Since this is an independent change, consider extracting it into a separate PR.